### PR TITLE
212 global column ordering dosent have t

### LINKED ISSUE #212: Global column ordering dosent have the dotted border and wrong spacing
### mantine-react-table version

v2-alpha

### react & react-dom versions

v18.0.2

### Describe the bug and the steps to reproduce it

Reorder column using global filter with this demo https://v2.mantine-react-table.com/docs/guides/column-ordering-dnd#enable-column-ordering-demo

### Minimal, Reproducible Example - (Optional, but Recommended)

-

### Screenshots or Videos (Optional)

_No response_

### Do you intend to try to help solve this bug with your own PR?

None

### Terms

- [X] I understand that if my bug cannot be reliably reproduced in a debuggable environment, it will probably not be fixed and this issue may even be closed.

### PR DESCRIPTION
**Pull request by alessandrojcm**
* Adds box shadow to table head when hovering.
* Fixes hovered and target state when dradding items from the global menu ordering item

Closes #212

### CODE REVIEW

**Comment by posabsolute** [comment]
@alessandrojcm is the 209 files change intended?

**Comment by alessandrojcm** [comment]
> @alessandrojcm is the 209 files change intended?

No, I don't know what is wrong with my fork is like it has not got the changes from this repo. Let me check.

**Comment by alessandrojcm** [comment]
Fixed it @posabsolute

**Review by posabsolute — APPROVED** [review]

File: packages/mantine-react-table/src/head/MRT_TableHeadRow.module.css [hunk 1]
```diff
@@ -4,6 +4,7 @@
     var(--mantine-color-dark-7)
   );
   display: table-row;
+  box-shadow: 0 rem(4) rem(8) var(--mantine-color-dark-2);
```
Comment (KevinVandy) [inline packages/mantine-react-table/src/head/MRT_TableHeadRow.module.css:7]: this box shadow does not look very good. removing

**Pull request merged**